x/leverage/keeper/store.go

```diff
@@ -0,0 +1,130 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/umee-network/umee/x/leverage/types"
+)
+
```

Comment:
This `store.go` file only contains the new un-exported get/set functions that were made for this feature, but we might consider moving more of them here in a separate PR.